app/views/application/your-choices/index.html

```diff
@@ -21,37 +21,56 @@
+          Select the option that applies to you
         </h1>
 
 
-        <p class="govuk-body-l">We want to make applying for compensation as easy as possible for you. We pay compensation for sexual assault or abuse to recognise the distress and mental trauma you have suffered.</p>
+        <p class="govuk-body-l">We process applications in different ways depending on what happened to you.</p>
 
         <h2 class="govuk-heading-m">
           Sexual assault or abuse
         </h2>
 
-        <p>We will:</p>
+        <p class="govuk-body">We can pay compensation:</p>
         <ul class="govuk-list govuk-list--bullet">
-          <li>ask the police to send us a report of the crime</li>
-          <li>not normally ask for your medical records</li>
-          <li>usually give you a decision within 3 months </li>
+          <li>for the nature of the sexual assault or abuse</li>
+          <li>to recognise the distress and mental trauma you have suffered</li>
         </ul>
 
+        <p class="govuk-body">We will normally base our decision on the information we get from the police and your application.</p>
+        <p class="govuk-body">We will usually make a decision within 4 months.</p>
+
         <h2 class="govuk-heading-m">
-          Sexual assault or abuse and additional injuries or losses
+          Sexual assault or abuse and other injuries or losses
         </h2>
 
-        <p>As well as a police report, we:</p>
+        <p class="govuk-body">We can pay compensation for:</p>
         <ul class="govuk-list govuk-list--bullet">
-          <li>will need information about your additional injuries or losses</li>
-          <li>will normally ask for your medical records</li>
-          <li>may arrange for an expert to diagnose your medical condition</li>
-          <li>will usually give you a decision within 12 months</li>
+          <li>financial losses, for example, if you were unable to work</li>
+          <li>physical injuries</li>
+          <li>pregnancy, loss of foetus or sexually transmitted disease</li>
+          <li>disabling mental injuries that are additional to the distress and mental trauma already suffered</li>
         </ul>
 
         <form class="form"  method="post">
 
+          <details class="govuk-details">
+            <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                What is a disabling mental injury?
+              </span>
+            </summary>
+            <div class="govuk-details__text">
+              A disabling mental injury is a condition that adversely affects your ability to perform routine tasks. These include work, social relationships and sexual relations.
+            </div>
+            <div class="govuk-details__text">
+              A medical expert must diagnose your disabling mental injury. It must last for at least 28 weeks to qualify for compensation over what we pay for distress and mental trauma.
```

Comment:
Change
_A medical expert must diagnose your disabling mental injury. It must last for at least 28 weeks to qualify for compensation over what we pay for distress and mental trauma._
To
_A medical expert must diagnose your disabling mental injury. It must last for at least 28 weeks to qualify for compensation over what we pay for **the** distress and mental trauma **caused by sexual assault or abuse**._